Rename 'Team Win' to 'TeamWin' in 'TWRP' topic (#3420)
github/explore: topics/twrp/index.md

---

> created_by: TeamWin

Reviewer: We can revert this also
```suggestion
created_by: Team Win
```

display_name: TWRP (software)

> short_description: Team Win Recovery Project (TWRP), pronounced "twerp", is an open-source software custom recovery image for Android-based devices.

Reviewer: Should we update here too?

Author: What do you mean?

Reviewer: ```suggestion
short_description: TeamWin Recovery Project (TWRP), pronounced "twerp", is an open-source software custom recovery image for Android-based devices.
```

Reviewer: Although I see on https://twrp.me/about/ that they refer to themselves as `Team Win`. So I'm not sure either of these changes are necessary.

Author: Yes, okay. Is there no way to redirect the 'Team Win' to the 'TeamWin' github profile?

Reviewer: Ok, if that’s the case you can do this:
```suggestion
short_description: Team Win Recovery Project (TWRP), pronounced "twerp", is an open-source software custom recovery image for Android-based devices.
github_url: https://github.com/TeamWin
```
